sdk/imagebuilder/arm-imagebuilder/test/imagebuilder_examples.ts

```diff
@@ -71,6 +75,14 @@ describe.skip("ImageBuilder test", () => {
     await recorder.stop();
   });
 
+  it.only("operations list test", async function () {
```

Comment:
skip other cases as no permission to do this operation https://github.com/Azure/azure-sdk-for-js/blob/44799b67c38d84d42410811fe077ac7b1031e3ad/sdk/imagebuilder/arm-imagebuilder/test/imagebuilder_examples.ts#L127